From: Takashi Iwai <tiwai@suse.de>
Subject: acpi_ec struct kABI compatibility
Patch-mainline: Never, kABI compatibility
References: bsc#1196213

After the drop of the recent ACPI patches (
commit 4a9af6cac050
  patches.suse/ACPI-EC-Rework-flushing-of-EC-work-while-suspended-t.patch
commit dc0075ba7f38
  patches.suse/ACPI-PM-s2idle-Cancel-wakeup-before-dispatching-EC-G.patch),
we need to re-add a couple of fields to struct acpi_ec back, although
it's basically internal (but unfortunately the type is exported).

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/acpi/internal.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/acpi/internal.h
+++ b/drivers/acpi/internal.h
@@ -183,6 +183,8 @@ struct acpi_ec {
 	struct work_struct work;
 	unsigned long timestamp;
 	unsigned long nr_pending_queries;
+	unsigned int events_in_progress;	/* FIXME: for kABI */
+	unsigned int queries_in_progress;	/* FIXME: for kABI */
 	bool busy_polling;
 	unsigned int polling_guard;
 };
